java/code_to_gast/java_main.py

```diff
@@ -6,18 +6,29 @@
 
 
 def java_to_gast(java_input):
-    input_ast = ''
+    '''
+    Input will be wrapped in a class and main function if no functions are present
+    If functions are present it will be wrapped in a class 
+    If a class declaration is given an error will be thrown 
+    '''
+    class_main_wrapped = '''
+    class Test {{
+        public static void artifical_wrapper_WkVHC(String[] args) {{
```

Comment:
Why suffix with WkVHC?

Reply:
just tried to pick a random string of characters to decrease the chance someone accidentally names their function this keyword